src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -61,37 +66,48 @@ public class HttpProtocolTests : HttpClientTest
         }
 
         [Theory]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.Netcoreapp)]
         [InlineData("HTTP/1.1 2345")]
         [InlineData("HTTP/A.1 200 OK")]
         [InlineData("HTTP/X.Y.Z 200 OK")]
-        // TODO #24713: The following pass on Windows on .NET Core but fail on .NET Framework.
-        //[InlineData("HTTP/0.1 200 OK")]
-        //[InlineData("HTTP/3.5 200 OK")]
-        //[InlineData("HTTP/1.12 200 OK")]
-        //[InlineData("HTTP/12.1 200 OK")]
-        // TODO #24713: The following pass on Windows on .NET Core but fail on UWP / WinRT.
-        //[InlineData("HTTP/1.1 200 O\nK")]
-        //[InlineData("HTTP/1.1 200OK")]
-        //[InlineData("HTTP/1.1 20c")]
-        //[InlineData("HTTP/1.1 23")]
-        //[InlineData("HTTP/1.1 2bc")]
-        // TODO #24713: The following pass on Windows but fail on CurlHandler on Linux.
-        //[InlineData("NOTHTTP/1.1")]
-        //[InlineData("HTTP/1.A 200 OK")]
-        //[InlineData("HTTP 1.1 200 OK")]
-        //[InlineData("ABCD/1.1 200 OK")]
-        //[InlineData("HTTP/1.1")]
-        //[InlineData("HTTP\\1.1 200 OK")]
-        //[InlineData("HTTP/1.1 ")]
-        //[InlineData("HTTP/1.1 !11")]
-        //[InlineData("HTTP/1.1 a11")]
-        //[InlineData("HTTP/1.1 abc")]
-        //[InlineData("HTTP/1.1 200 O\rK")]
-        //[InlineData("HTTP/1.1\t\t")]
-        //[InlineData("HTTP/1.1\t")]
-        //[InlineData("HTTP/1.1  ")]
-        //[InlineData("NOTHTTP/1.1 200 OK")]
+        // The following pass on Windows on .NET Core but fail on .NET Framework.
+        [InlineData("HTTP/1.12 200 OK")]
+        [InlineData("HTTP/12.1 200 OK")]
+        // The following pass on Windows on .NET Core but fail on UWP / WinRT.
+        [InlineData("HTTP/1.1 200 O\nK")]
+        [InlineData("HTTP/1.1 200OK")]
+        [InlineData("HTTP/1.1 20c")]
+        [InlineData("HTTP/1.1 23")]
+        [InlineData("HTTP/1.1 2bc")]
         public async Task GetAsync_InvalidStatusLine_ThrowsException(string responseString)
+        {
+            await GetAsyncThrowsExceptionHelper(responseString);
+        }
+
+        [PlatformSpecific(TestPlatforms.Windows)]
+        [Theory]
+        // The following pass on Windows but fail on CurlHandler on Linux.
```

Comment:
Unfortunately the way the tests are set up right now, the managed handler tests complicate platform specific, but as this is written, this test won't run with managed handler on Linux. Probably the platform specific attribute should be removed and imperative code should be used in the test to bail early if it's looking and not managed handler.

Reply:
Will fix.